genestack_client/files_util.py

```diff
@@ -380,6 +381,39 @@ def share_files(self, accessions, group, destination_folder=None, password=None)
         if destination_folder is not None:
             share_utils.invoke('linkFiles', accessions, destination_folder, group)
 
+    def share_folder(self, folder_accession, group, destination_folder=None, password=None):
+        """
+        Recursively share folder.
+
+        :param folder_accession: accession of the folder
+        :type folder_accession: basestring
+        :param group: accession of the group to share the files with
+        :param destination_folder: accession of folder to link shared folder into.
+               No links are created if ``None``.
+        :type destination_folder: str
+        :param password: password for sharing,
+               if not specified, will be asked for in an interactive prompt (if possible)
+        :type: str
+        :rtype: None
+        """
+        self.share_files([folder_accession], group, destination_folder=destination_folder, password=password)
+        share_utils = self.connection.application('genestack/shareutils')
+        limit = 100
+        delay_seconds = 1  # delay between attempts
+
+        offset = 0
+        while True:
```

Comment:
Can you add some comments here about the sharing procedure too? It's quite tedious to look it up in the `shareChunkInFolder` documentation

Reply:
Done